guides/source/caching_with_rails.md

```diff
@@ -3,12 +3,25 @@
 Caching with Rails: An Overview
 ===============================
 
-This guide is an introduction to speeding up your Rails app with caching.
+This guide is an introduction to speeding up your Rails application with caching.
+
+Caching means to store the content generated during the request-response cycle
```

Comment:
"the content" makes it feel like it's the whole response body, instead of merely part of it (as it could be with fragment caching). I would `the` before content.